### PR TITLE
EPMDJ-3142: Add new "Manage Session" panel

### PR DESCRIPTION
**Pull request by RomanAchilov**
Issue: EPMDJ-3142

### CODE REVIEW

**Review by pavyarov — CHANGES_REQUESTED** [review]
I have coupe of questions:
1. Why you managing error messages at frontend?

File: src/modules/manage-sessions-pane/manage-active-sessions/session-info/session-info.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+export const SessionInfo = sessionInfo(
+  ({
+    className, testType, sessionId, agentId, abortingSessionIdByAgentId, setAbortingSessionIdByAgentId, showGeneralAlertMessage,
+  }: Props) => {
+    const { pluginId = '' } = useParams<{ pluginId: string;}>();
+    const disabled = Boolean(abortingSessionIdByAgentId && sessionId + agentId !== abortingSessionIdByAgentId);
+    return (
+      <div className={className}>
+        {abortingSessionIdByAgentId === sessionId + agentId ? (
```
Comment (pavyarov) [inline src/modules/manage-sessions-pane/manage-active-sessions/session-info/session-info.tsx:33]: too much logic, split it to component

File: src/modules/manage-sessions-pane/manage-active-sessions/session-info/session-info.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+const sessionInfo = BEM(styles);
+
+export const SessionInfo = sessionInfo(
+  ({
+    className, testType, sessionId, agentId, abortingSessionIdByAgentId, setAbortingSessionIdByAgentId, showGeneralAlertMessage,
+  }: Props) => {
+    const { pluginId = '' } = useParams<{ pluginId: string;}>();
+    const disabled = Boolean(abortingSessionIdByAgentId && sessionId + agentId !== abortingSessionIdByAgentId);
```
Comment (pavyarov) [inline src/modules/manage-sessions-pane/manage-active-sessions/session-info/session-info.tsx:30]: `sessionId + agentId !== abortingSessionIdByAgentId` what?

File: src/modules/manage-sessions-pane/manage-active-sessions/session-info/session-info.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                    size="small"
+                    onClick={() => { setAbortingSessionIdByAgentId(sessionId + agentId); }}
+                    disabled={disabled}
+                  >
+                    Abort
+                  </LinkButton>
+                  <Button
+                    type="secondary"
+                    size="small"
+                    onClick={() => finishSession(agentId, sessionId, pluginId, showGeneralAlertMessage)}
```
Comment (pavyarov) [inline src/modules/manage-sessions-pane/manage-active-sessions/session-info/session-info.tsx:67]: to much arguments

File: src/modules/manage-sessions-pane/manage-sessions-pane.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+export const ManageSessionsPane = manageSessionsPane(
+  ({
+    className, isOpen, onToggle,
+  }: Props) => {
+    const [isNewSession, setIsNewSession] = React.useState(false);
+    const { generalAlertMessage, showGeneralAlertMessage } = useGeneralAlertMessage();
+    const {
+      agentId = '', serviceGroupId = '', pluginId = '', buildVersion = '',
+    } = useParams<{ agentId: string; serviceGroupId: string; pluginId: string; buildVersion: string}>();
+    const agentType = serviceGroupId ? 'ServiceGroup' : 'agent';
```
Comment (pavyarov) [inline src/modules/manage-sessions-pane/manage-sessions-pane.tsx:47]: why agent is't capitalized?

File: src/modules/manage-sessions-pane/use-active-sessions.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+import { defaultTest2CodePluginSocket } from 'common/connection';
+
+export function useActiveSessions<Data>(agentType: string, id: string, buildVersion?: string) {
```
Comment (pavyarov) [inline src/modules/manage-sessions-pane/use-active-sessions.ts:5]: any reason for generic here?

File: src/pages/agent-full-page/coverage-plugin/overview/active-scope-info/active-scope-info.tsx [hunk 6]
```diff
@@ -39,9 +39,9 @@ export const ActiveScopeInfo = activeScopeInfo(({
 
   const menuActions = [
     {
-      label: 'Manage sessions',
+      label: 'Manage Sessions',
       icon: 'ManageSessions',
-      onClick: () => dispatch(openModal('ManageSessionsModal', null)),
+      onClick: () => dispatch(openModal('ManageSessionsPane', null)),
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/overview/active-scope-info/active-scope-info.tsx:44]: so we have panels but this particular case is pane?

**Review by pavyarov — CHANGES_REQUESTED** [review]

File: src/modules/manage-sessions-pane/manage-active-sessions/session-info/session-info.tsx (hunk 2 above)
Comment (RomanAchilov) [reply src/modules/manage-sessions-pane/manage-active-sessions/session-info/session-info.tsx:30]: I need this to correctly disable sessions in the service group

**Review by pavyarov — APPROVED** [review]

**Pull request merged**